packages/jest-reporters/src/GitHubActionsReporter.ts

```diff
@@ -81,4 +129,275 @@ export default class GitHubActionsReporter extends BaseReporter {
       `\n::${type} file=${file},line=${line},title=${title}::${message}`,
     );
   }
+
+  __isLastTestSuite(results: AggregatedResult): boolean {
```

Comment:
instead of all these `__` methods, just use `private` modifier. They don't exist on the other reporters for legacy reasons, but these are new methods, so hiding them (in TS at least) is fine

Reply:
Fixed